test/images/agnhost/agnhost.go

```diff
@@ -51,7 +51,7 @@ import (
 func main() {
 	rootCmd := &cobra.Command{
 		Use:     "app",
-		Version: "2.27",
+		Version: "2.28",
```

Comment:
Same as above

Reply:
I think is correct, this has to be bumped first so we can promote it later